dist/typeahead.bundle.js

```diff
@@ -1206,7 +1206,7 @@
                     that.highlight && highlight({
                         className: "tt-highlight",
                         node: $suggestions[0],
-                        pattern: query
+                        pattern: _.isFunction(that.highlight) ? that.highlight(query) : query
```

Comment:
Would call with signature of `that.highlight(query, suggestions)` as the results may not correlate directly with the query, for instance misspellings corrected with Levenshtein distance.